### PR TITLE
wd env bugfix and some clean code

### PR DESCRIPTION
**Pull request by lin755**

### CODE REVIEW

File: wd_util.c [hunk 1]
```diff
@@ -312,12 +312,76 @@ void dump_env_info(struct wd_env_config *config)
...
+		return;
+
+	FOREACH_NUMA(i, config, config_numa)
+		free(config_numa->dev);
+
+	free(config->config_per_numa);
+}
+
+static void wd_get_dev_numa(struct uacce_dev_list *head,
+			    int *numa_dev_num, int *numa_num)
```
Comment (hzhuang1) [inline wd_util.c:345]: Could you help to share more comments on **numa_dev_num** and **numa_num**?

File: wd_util.c [hunk 2]
```diff
@@ -312,12 +312,76 @@ void dump_env_info(struct wd_env_config *config)
...
+{
+	struct uacce_dev_list *list = head;
+
+	while (list) {
+		if (list->dev->numa_id < 0)
+			list->dev->numa_id = 0;
+
+		if (!numa_dev_num[list->dev->numa_id])
+			(*numa_num)++;
```
Comment (hzhuang1) [inline wd_util.c:354]: You increased **\*numa_num** directly. But what's the initial/expected value of **\*numa_num**?

File: wd_util.c (hunk 1 above)
Comment (lin755) [reply wd_util.c:345]: numa_dev_num represents how many device of one type(like sec2 support cipher) on this numa, numa_num represents how many numa has such devices. 
For example, if we are on kunpeng 920, only node 0 and node 2 has sec2 devices,but node 0 may connect with two  sec2 devices,so numa_dev_num is 2, not 1.

Comment (hzhuang1) [reply wd_util.c:345]: Good. Please comment it in the code.

File: wd_util.c (hunk 2 above)
Comment (lin755) [reply wd_util.c:354]: You can see numa_num initial value is 0 in wd_alloc_numa statement. Add it one time only when we find a device on a new numa node.if device numa node is 0, 0, 2，we add it one only in first and third device

File: wd_util.c (hunk 1 above)
Comment (lin755) [reply wd_util.c:345]: ok,I will change it later

File: wd_util.c (hunk 2 above)
Comment (hzhuang1) [reply wd_util.c:354]: It seems that wd_get_dev_numa() is only called once. And **\*numa_num** is initialized out of the wd_get_dev_numa(). Maybe you can initialize it just in wd_get_dev_numa().

Comment (lin755) [reply wd_util.c:360]: Resubmit these changes now.

**Pull request merged**